aif360/metrics/mdss_classification_metric.py

```diff
@@ -0,0 +1,68 @@
+from collections import defaultdict
+from aif360.datasets import BinaryLabelDataset
+from aif360.metrics import ClassificationMetric
+
+from aif360.metrics.mdss.ScoringFunctions import Bernoulli
+from aif360.metrics.mdss.MDSS import MDSS
+
+import pandas as pd
+
+class MDSSClassificationMetric(ClassificationMetric):
+    """
+        Bias subset scanning is proposed as a technique to identify bias in predictive models using subset scanning [1].
+        This class is a wrapper for the bias scan scoring and scanning methods that uses the ClassificationMetric abstraction.
+    References:
+        .. [1] Zhang, Z., & Neill, D. B. (2016). Identifying significant predictive bias in classifiers. arXiv preprint arXiv:1611.08292.
+    """
+    def __init__(self, dataset: BinaryLabelDataset, classified_dataset: BinaryLabelDataset, 
+                scoring_function: Bernoulli, unprivileged_groups: dict = None, privileged_groups:dict = None):
+    
+        super(MDSSClassificationMetric, self).__init__(dataset, classified_dataset,
+                                                       unprivileged_groups=unprivileged_groups,
+                                                       privileged_groups=privileged_groups)
+
+        self.scanner = MDSS(scoring_function)
+    
+    def score_groups(self, privileged=True, penalty = 0.0):
+        """
+        compute the bias score for a prespecified group of records.
```

Comment:
can you state briefly what privileged=True vs False means (direction of bias). Also a line on how the penalty term impacts the returned groups?

Reply:
Done